.buildkite/scripts/steps/cloud/deploy.json

```diff
@@ -141,7 +141,7 @@
               ],
               "id": "ml",
               "size": {
-                "value": 1024,
+                "value": 4096,
```

Comment:
Temporarily bumping cloud deploy ML node capacity to 4GB so ELSER can be tested. Please revert this once testing is complete.